### DISCUSSION TITLE
Toggle Save Result to prevent overflow of "Failed to save result" errors

### DISCUSSION THREAD

**Discussion by ghost** [monkeytypegame/monkeytype#4976] [Ideas]
If I could toggle saving results to off, it would prevent the below and just be super useful.

![image](https://github.com/monkeytypegame/monkeytype/assets/46577890/c958f0ae-0724-49af-b9eb-4a18e9f9ba30)

**Comment by Miodec** [comment]
Esc > Result saving > off
